hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -552,7 +552,7 @@ async def submit(self,
 @asyncinit
 class BatchClient:
     async def __init__(self, billing_project, deploy_config=None, session=None,
-                       headers=None, _token=None):
+                       headers=None, _token=None, _token_file=None):
```

Comment:
same thoughts here. I also don't think _token is used. We can remove it if so.

Reply:
`_token` is used in one of the tests.